master/master.cfg

```diff
@@ -162,6 +162,11 @@ extra_factory_args = {
         # test curses as well
         "test_with_PTY": True,
     },
+    "cstratak-RHEL8-ppc64le": {
+        # Increase the timeout on specific branches
+        "slow_branches": ['3.8', '3.9', '3.10'],
```

Comment:
It would be simpler to increase the timeout on all branches, no?

Reply:
It would, but do we want that?

Reply:
I'm not sure why, but this specific worker is slower than other worker. It got a few timeout issues. I suggest to increase the timeout. I don't see any downside. It's just a hard limit.

Reply:
OK.
I didn't find a pre-existing setting for that either, so I added one the same way as `slow_branches` before.
Does this look good?